src/components/ProcessGraph.tsx

```diff
@@ -91,6 +108,10 @@ const ProcessGraph: React.FC<ProcessGraphProps> = ({ nodes, edges, hideZoomButto
     setPopupTargetNode(undefined);
   };
 
+  const closeInfoPopup = () => {
+    setInfoVisible(false);
+  };
+
```

Comment:
Could just use `setInfoVisible(false)` directly instead of wrapping it into a function that does not do anything else